Extensions/PlatformBehavior/Extension.cpp

```diff
@@ -501,6 +514,81 @@ void DeclarePlatformBehaviorExtension(gd::PlatformExtension& extension) {
         .SetFunctionName("IgnoreDefaultControls")
         .SetIncludeFile("PlatformBehavior/PlatformerObjectRuntimeBehavior.h");
 
+    aut.AddAction("CanGrabPlatforms",
+                  _("Can Grab Platform"),
+                  _("De/activate when the platformer object can grab the platform."),
+                  _("_PARAM0_ can grab the platforms: _PARAM2_"),
+                  _("Controls"),
+                  "res/conditions/keyboard24.png",
+                  "res/conditions/keyboard.png")
+        .AddParameter("object", _("Object"))
+        .AddParameter("behavior", _("Behavior"), "PlatformerObjectBehavior")
+        .AddParameter("yesorno", _("Can grab the platfroms"))
+        .MarkAsAdvanced()
+        .SetFunctionName("SetCanGrabPlatforms")
+        .SetIncludeFile("PlatformBehavior/PlatformerObjectRuntimeBehavior.h");
```

Comment:
If the function is not implemented in C++, don't call SetFunctionName and don't call SetIncludeFile. In the hypothesis of GDCpp being added back to the editor, the action without an associated function won't be shown in the editor.

Reply:
SetCanGrabPlatforms is in C++ here
https://github.com/4ian/GDevelop/blob/master/Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.cpp#L501

Reply:
Nevermind it's deleted in the lastest commit.